aspnet/aspnet/overview/developing-apps-with-windows-azure/building-real-world-cloud-apps-with-windows-azure/automate-everything.md

```diff
@@ -168,7 +168,7 @@ The database creation script retrieves the dev machine's IP address and sets a f
 
     [!code-powershell[Main](automate-everything/samples/sample14.ps1)]
 
-The Fix It app uses separate membership and application databases. It's also possible to put both membership and application data in a single database. For an example that uses a single database, see [Create an ASP.NET MVC app with auth and SQL DB and deploy to Azure App Service](https://www.windowsazure.com/en-us/develop/net/tutorials/web-site-with-sql-database/).
+The Fix It app uses separate membership and application databases. It's also possible to put both membership and application data in a single database. For an example that uses a single database, see [Create an ASP.NET MVC app with auth and SQL DB and deploy to Azure App Service](https://docs.microsoft.com/aspnet/core/security/authorization/secure-data).
```

Comment:
I think this link should be replaced with a link to MVC 5/Azure security topic (Or deleted like I did few hours ago!).